pkg/runner/run_context.go

```diff
@@ -202,8 +203,10 @@ func (rc *RunContext) startJobContainer() common.Executor {
 			rc.JobContainer.Start(false),
 			rc.JobContainer.UpdateFromImageEnv(&rc.Env),
 			rc.JobContainer.UpdateFromEnv("/etc/environment", &rc.Env),
-			rc.JobContainer.Exec([]string{"mkdir", "-m", "0777", "-p", ActPath}, rc.Env, "root", ""),
+			rc.JobContainer.Exec([]string{"mkdir", "-m", "0777", "-p", ActPath}, rc.Env, "0", ""),
 			rc.JobContainer.CopyDir(copyToPath, rc.Config.Workdir+string(filepath.Separator)+".", rc.Config.UseGitIgnore).IfBool(copyWorkspace),
+			rc.JobContainer.Exec([]string{"chown", "-R", rc.Config.ContainerUser, copyToPath}, rc.Env, "root", "").IfBool(copyWorkspace),
+			rc.JobContainer.Exec([]string{"chown", "-R", rc.Config.ContainerUser, "/home/runner"}, rc.Env, "root", ""),
```

Comment:
```suggestion
			rc.JobContainer.Exec([]string{"chown", "-R", rc.Config.ContainerUser, copyToPath}, rc.Env, "0", "").IfBool(copyWorkspace),
			rc.JobContainer.Exec([]string{"chown", "-R", rc.Config.ContainerUser, "/home/runner"}, rc.Env, "0", ""),
```
In case image doesn't have `root`

Reply:
Thanks!